test_settings:
  seed: 40
  base_file_name: "td7_agent"
  save_date: "20241019"
  max_episode_steps: 500
  use_checkpoints: true

start_goal_pairs:
- start: {x: 4.0, y: 1.0, theta: 3.141593}
  goal: {x: -5.0, y: -3.0}
- start: {x: -5.0, y: -1.0, theta: 0}
  goal: {x: 5.0, y: 5.0}
- start: {x: 5.0, y: -2.0, theta: 3.141593}
  goal: {x: -5.0, y: 3.0}
- start: {x: -4.0, y: 5.0, theta: 0}
  goal: {x: 5.0, y: -3.0}
- start: {x: -4.0, y: -1.0, theta: 0}
  goal: {x: 5.0, y: 3.0}
# Waipoints for mapping
# - start: {x: 0.0, y: 0.0, theta: 0}
#   goal: {x: 5.0, y: 3.0}
# - start: {x: 5.0, y: 3.0, theta: 3.141593}
#   goal: {x: 0.0, y: 0.0}
# - start: {x: 0.0, y: 0.0, theta: 0}
#   goal: {x: -5.0, y: 3.0}
# - start: {x: -5.0, y: 3.0, theta: 0}
#   goal: {x: 0.0, y: 0.0}
# - start: {x: 0.0, y: 0.0, theta: 0}
#   goal: {x: -5.0, y: -3.0}
# - start: {x: -5.0, y: -3.0, theta: 0}
#   goal: {x: 0.0, y: 0.0}
# - start: {x: 0.0, y: 0.0, theta: 0}
#   goal: {x: 5.0, y: -3.0}
  